content/doc/book/managing/plugins.adoc

```diff
@@ -4,13 +4,261 @@ layout: section
 :notitle:
 :description:
 :author:
-:email: jenkinsci-users@googlegroups.com
+:email: jenkinsci-docs@googlegroups.com
 :sectanchors:
-:toc: left
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
 
 = Managing Plugins
 
+////
+Pages to mark as deprecated by this document:
+
+https://wiki.jenkins-ci.org/display/JENKINS/Plugins (header)
+https://wiki.jenkins-ci.org/display/JENKINS/Removing+and+disabling+plugins
+https://wiki.jenkins-ci.org/display/JENKINS/Pinned+Plugins
+////
+
+Plugins are the primary means of enhancing the functionality of a Jenkins
+environment to suit organization or user-specific needs. There are
+link:https://plugins.jenkins.io[over a thousand different plugins]
+which can be installed on a Jenkins master and integrate various
```

Comment:
'to integrate'
'and much more.' [more grammatically correct, even if it sounds like an infomercial]